pinot-core/src/main/java/org/apache/pinot/core/data/manager/BaseTableDataManager.java

```diff
@@ -211,4 +211,7 @@ private void closeSegment(SegmentDataManager segmentDataManager) {
   public String getTableName() {
     return _tableNameWithType;
   }
+
+  @Override
+  public String getTableDataDir() { return _tableDataDir; }
```

Comment:
Could you try formatting this? Should not be in the same line.

Reply:
done.